packages/client/pages/create-profile/index.tsx

```diff
@@ -149,11 +149,11 @@ export default function CreateProfile() {
       m="auto"
       minHeight="100vh"
       h="100%"
-      width={{ '2xl': '70%', md: '90%', sm: '100%' }}
+      width={{ '2xl': '70%', md: '90%', base: '100%' }}
       pt={{ '2xl': '2%', lg: '10%', md: '15%', sm: '17%', base: '15%' }}
       pb={{ '2xl': '10%', lg: '10%', md: '15%', sm: '17%', base: '20%' }}
-      pl={{ '2xl': '5%', lg: '22.5%', sm: '5%', base: '5%' }}
-      pr={{ sm: '3%', base: '3%' }}
```

Comment:
Since we've defined a value of `3%` for _all_ breakpoints (due to fallback), we don't need to use responsive object syntax at all. (Type 3)